daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageVersion.scala

```diff
@@ -83,9 +94,8 @@ object LanguageVersion {
 
   // All the versions
   val DevVersions: VersionRange[LanguageVersion] =
-    EarlyAccessVersions.copy(max = v1_dev)
+    EarlyAccessVersions.copy(max = v2_dev)
```

Comment:
"DevVersions" makes me think `1.dev` and `2.dev` but the comment (and the code) mean this includes all versions, we should rename accordingly

Reply:
Looking at EarlyAccessVersions, it looks like the convention in this file is that version ranges are named after the delta wrt to the previous version range. I agree it is misleading but I am tempted to keep the names that way to minimize this PR: if I change DevVersions, I'll need to change EarlyAccessVersions too, and this will lead to noise for the code reviewer (who I feel for already). We'll have to revamp all of this as soon as version ranges disappear anyway.